changelog.md

```diff
@@ -2,7 +2,9 @@
 
 ## UNRELEASED
 
-_Nothing yet_
+**Added:**
```

Comment:
Make this "fixed" as this is a patch increment. The term fixed might be a bit confusing in this context, but that'll have to do.